### PR TITLE
F# System.ToString code examples

### PR DESCRIPTION
**Pull request by ilyakharlamov**
# Title

F# System.ToString code examples

## Summary

Added F# code examples for System.ToString method

## Details

The F# code examples for System.ToString method are missing at https://msdn.microsoft.com/en-us/library/ms131011(v=vs.110).aspx?cs-save-lang=1&cs-lang=fsharp#code-snippet-1 for some reason.
So I have added the F# samples for this method so that the output would be exactly same as VB/C#.

## Suggested Reviewers

@yishengjin1413 
@cartermp

### CODE REVIEW

**Review by svick — COMMENTED** [review]
Note that I'm reviewing just the code, I don't know if this is the right way to add a new language.

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.convert.tostring2/fs/tostring2.fs [hunk 1]
```diff
@@ -0,0 +1,172 @@
+open System
+
+let convertByte =
+//<snippet9>
+    let bases = [2; 8; 10; 16]
+    let (numbers:byte list) = [ Byte.MinValue; 12uy; 103uy; Byte.MaxValue ]
```
Comment (ilyakharlamov) [inline samples/snippets/fsharp/VS_Snippets_CLR_System/system.convert.tostring2/fs/tostring2.fs]: Just wanted to be explicit but you are right, where type inference works it's better to omit that.

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.convert.tostring2/fs/tostring2.fs [hunk 2]
```diff
@@ -0,0 +1,172 @@
+open System
+
+let convertByte =
```
Comment (ilyakharlamov) [inline samples/snippets/fsharp/VS_Snippets_CLR_System/system.convert.tostring2/fs/tostring2.fs]: Just wasn't sure what's the best way to convert a C# code with side-effects that prints to the console. There are three options I see: 
1. make it pure (the functional way) but print in the main
2. keep it as close to C# as possible with side effects ignoring the functional specific i.e. make it (unit -> unit) or just (unit) both are not ideal.

**Comment by ilyakharlamov** [comment]
Withdraw and will make another pull request

**Comment by svick** [comment]
> Withdraw and will make another pull request

By the way, there is no reason to do that. If you update the branch of your fork of the repo (like you already did twice in https://github.com/dotnet/docs/pull/2641), it will automatically update the Pull Request.

And it's fine to keep your Pull Request open while it's waiting for you to make some changes.

**Pull request closed without merging**